students/mimalvarez_pintor/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Apr 23 22:22:44 2020
+
+@author: miriam
+"""
+
+#Slicing Lab
+def exchange_first_last(seq):
+    return(seq[-1:]+seq[1:-1]+seq[:1])
+#print(exchange_first_last(a_string))
+#print(exchange_first_last(a_tuple))
+
+def every_other_rmv(seq):
+    return(seq[0::2])
+#print(every_other_rmv(a_string))
+#print(every_other_rmv(a_tuple))
+
+def FirstLastFourRmv_EveryOther(seq):
```

Comment:
make sure to use snake_case function naming -> `def first_last_four_remove_every_other`